vnext/Microsoft.ReactNative/Views/ReactRootControl.cpp

```diff
@@ -607,6 +608,53 @@ void ReactRootControl::ReloadViewHost() noexcept {
   }
 }
 
+void ReactRootControl::AttachBackHandlers(XamlView const &rootView) noexcept {
+  auto rootElement(rootView.as<winrt::UIElement>());
+  if (rootElement == nullptr) {
```

Comment:
`as` will already throw if we can't QI (i.e. we terminate since it hits the noexcept boundary).

Reply:
The non-throwing version is the try_as.

---
In reply to: [409825498](https://github.com/microsoft/react-native-windows/pull/4623#discussion_r409825498) [](ancestors = 409825498)

Reply:
Switched to `try_as` with a null check

Reply:
@vmoroz was there a preference towards null-checking, or was pointing towards try_as more educational?

It's splitting hairs but I actually slightly preferred the terminating version.